h2o-docs/src/product/data-science/algo-params/distribution.rst

```diff
@@ -80,6 +80,87 @@ Some examples of response distributions are provided below.
++-------------------+-------------------------------------------------------------------------------------------+
+| Quasibinomial     | - :math:`y==f:0`                                                                          |
+|                   | - :math:`f>1: -2w*y*log(f)`                                                               |
+|                   | - :math:`f<0: -2w*(1-y)*log(1-f) \text{ else}`                                            |
+|                   | - :math:`-2w*(y*log(f)+(1-y)*log(1-f))`                                                   |
++-------------------+-------------------------------------------------------------------------------------------+
+| Gaussian          | - Equivalent to wMSE (weighted mean squared error):                                       |
+|                   | - :math:`w*(y-f)^2`                                                                       |
++-------------------+-------------------------------------------------------------------------------------------+
+| Poisson           | - With :math:`f=link(f)`                                                                  |
+|                   | - :math:`-2w*(y*f-exp(f))`                                                                |
++-------------------+-------------------------------------------------------------------------------------------+
+| Gamma             | - With :math:`f=link(f)`                                                                  |
+|                   | - :math:`2w*(y*exp(-f)+f)`                                                                |
++-------------------+-------------------------------------------------------------------------------------------+
+| Laplace           | :math:`w*\mid{y-f}\mid`                                                                   |
++-------------------+-------------------------------------------------------------------------------------------+
+| Quantile          | - :math:`y>f: w*QuantileAlpha*(y-f)`                                                      |
+|                   | - :math:`y<=f: w*1-QuantileAlpha*(f-y)`                                                   |
++-------------------+-------------------------------------------------------------------------------------------+
+| Huber             | - :math:`\mid{y-f}\mid <= \text{Huber_Delta}`:                                            |
+|                   | - :math:`w*(y-f)^2` (Equivalent to wMSE)                                                  |
+|                   | - :math:`y<=f`:                                                                           |
+|                   | - :math:`2w*(\mid{y-f}\mid - \text{Huber_Delta})*\text{Huber_Delta}` (Equivalent to wMAE) |
++-------------------+-------------------------------------------------------------------------------------------+
+| Modified_Huber    | - with :math:`yf=(2y-1)*f`                                                                |
```

Comment:
I don't really like this term `yf`. It can confuse you with y*f. I would prefer giving it a new name. @maurever can this have a designated name? Does it have a mathematical name for this? Or can we just call it a new letter?

Reply:
Good point. I am not sure it has some mathematical name. We can call it a new letter, for example, `z`...